docs/pipeline-api.md

```diff
@@ -5958,9 +5958,9 @@ RunStatus
 <h3 id="tekton.dev/v1alpha1.VerificationPolicy">VerificationPolicy
 </h3>
 <div>
-<p>VerificationPolicy defines the Tekton resources and corresponding authorities to verify.
-The VerificationPolicy is used in trusted resources to store the public keys to verify
-Tekton resources.</p>
+<p>VerificationPolicy defines the rules to verify Tekton resources.
```

Comment:
It seems like our CI is not verifying that the pipeline api doc is getting generated correctly :(
Ideally this would be a separate PR

Reply:
Added https://github.com/tektoncd/pipeline/pull/5939.